### PR TITLE
Godwin: adopt-a-dog

### PR DESCRIPTION
**Pull request by Headieh**
HTML code and some css for the mockups for assignment 1 (lesson1 assignment - 
852c13a)

### CODE REVIEW

**Comment by Headieh** [comment]
HTML code and css for the mockups for lesson 2 assignment 3 (lesson2 assignment3 - 
0a6b9e2)

File: scripts/adopt.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+function disableButton(id) {
+	document.getElementById(id).disabled = true;
```
Comment (MadEste) [inline scripts/adopt.js:2]: Nice concept to keep the user from pressing adopt multiple times on the same dog!

File: scripts/adopt.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+let eachFee = [];
+
+function adopt(x) {
+	if (x === 'murphy') {
```
Comment (MadEste) [inline scripts/adopt.js:28]: Why not pass the fee instead of name, and avoid the if statements?

File: scripts/about.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
+function about(x) {
```
Comment (MadEste) [inline scripts/about.js]: This works well, but remember you can also pass more than one parameter and avoid the hardcoded if statements.

File: styles/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+}
+
+a {
+	color: rgb(81, 156, 183);
+	font-weight: bold;
+	display: block;
+	text-decoration: none;
+}
+
+a:hover,
```
Comment (MadEste) [inline styles/stylesheet.css:159]: I believe this is causing some issues with your menu items "Jitter" on rollover.

File: scripts/checkout.js [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+	info.push($("#city").val());
+	info.push($("#state").val());
+	info.push($("#zip").val());
+	info.push($("input:radio[name='New']:checked").val());
+	info.push($("#location").val());
+	console.log(info);
+	alert("Thank you. The form information has been received");
+}
+
+$('form').on('submit', submitEvent);
```
Comment (MadEste) [inline scripts/checkout.js:18]: Would prefer you accomplish this without JQuery for assignment 1, but everything looks good!

**Pull request closed without merging**